chalice/deploy/deployer.py

```diff
@@ -141,15 +141,29 @@ def confirm(self, text, default=False, abort=False):
         # type: (str, bool, bool) -> bool
         return default
 
+    def Abort(self):
```

Comment:
So the existing prompter is basically used for control flow around which resources are deployed/deleted. I think rather than raising an exception or aborting outright I could also just add an `else` clause and move the actual resource deletion into that. And then take the abort method off of the Prompter interface.